src/urh/signalprocessing/Signal.py

```diff
@@ -131,6 +135,27 @@ def __load_wav_file(self, filename: str):
 
         self.sample_rate = sample_rate
 
+    def __load_sub_file(self, filename: str):
+        # Flipper RAW file format (OOK): space separated values, number of samples above (positive value -> 1)
+        # or below (negative value -> 0) center
+        params = {"min": 0, "max": 255, "fmt": np.uint8}
+        params["center"] = (params["min"] + params["max"]) / 2
+        arr = []
+        with open(filename, 'r') as subfile:
+            for line in subfile:
+                dataline = re.match(r'RAW_Data:\s*([-0-9 ]+)\s*$', line)
+                if dataline:
```

Comment:
Shouldn't we rather check
```suggestion
                if len(dataline) > 1:
```
as we refer to `dataline[1]` in the next line?

Reply:
When the re fails, dataline does not have a len. This code follows the concept from: https://gist.github.com/jinschoi/40a470e432c6ac244be8159145454b5c